ht/cogs/roll.py

```diff
@@ -45,6 +49,60 @@ async def armiger(self, ctx, user: converters.Armiger = None):
 		await self.add_rolls(embed, "AND NOT personal", user, "Artist gallery")
 		await ctx.send(embed = embed)
 
+	@commands.command(
+		help = "Looks up the symbology of a user's coat of arms.\nUses GreiiEquites' https://roll-of-arms.com as a source.",
+		aliases = ("s", "symbols")
+	)
```

Comment:
```suggestion
	)
	@utils.trigger_typing
```
Oh also I forgot, commands that take a while should include this decorator. It makes the bot show the "Heraldtron is typing..." thing which means the user isn't confused about it not doing stuff immediately.